assets/styles.css

```diff
@@ -0,0 +1,202 @@
+/* Apple Clone Website Styles */
+
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400&display=swap");
+
+/* CSS RESET */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  align-items: center;
+}
+
+h3 {
+  margin: 20px auto;
+}
+
+.container {
+  width: 100%;
+  max-width: 1440px;
+  margin: auto;
+  font-family: "Lato", sans-serif;
+}
+
+.flex {
+  display: flex;
+}
+
+.color-white {
+  color: #fff;
+}
+
+.hero {
+  min-height: 77vh;
+  background-image: url("./images/your_verse_hero.jpg");
+  background-repeat: no-repeat;
+  background-position: 50% 100%;
+  background-size: 1440px 1315px;
```

Comment:
- [ ] Your implementation of background size looks good, but remember that you can use `background-size: cover; ` so it is more general and you don't have to specify a size:

![image](https://user-images.githubusercontent.com/53256139/88645292-91985d00-d089-11ea-8282-338526d80fc8.png)